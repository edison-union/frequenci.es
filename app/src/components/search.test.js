import React from 'react'
import renderer from 'react-test-renderer'
import Search from './search'
import { countries } from '../tests/setup'
import wrapWithProvider from '../../wrap-with-provider'

describe("Search", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Search
      countries={countries.map((country) => {
       return {
         name: country.name,
         code: country.country.toLowerCase(),
         airports: country.airports.length
       }
     })} childActive={0}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
