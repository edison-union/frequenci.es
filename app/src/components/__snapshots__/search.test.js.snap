// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<section
  className="sc-bdVaJa fKqyOz"
>
  <h4
    className="sc-bwzfXH jQPHvi"
  >
    Search
  </h4>
  <input
    className="sc-gzVnrw eItwKo"
    onKeyUp={[Function]}
  />
</section>
`;
